statsmodels/stats/tests/test_lilliefors.py

```diff
@@ -0,0 +1,26 @@
+from numpy.testing import assert_, assert_raises
+from scipy import stats
+import nose
+from statsmodels.stats._lilliefors import lilliefors
+import numpy as np
+
+class TestLilliefors(object):
```

Comment:
Understood, I'll get reference values from the R implementation and write new tests. Thanks for your guidance!